### PR TITLE
Mobile product listing tweaks 2

### PR DESCRIPTION
**Pull request by mkllnk**
#### What? Why?

Follow up of #5968.

Fixing issues in the mobile product listing branch found by Jen last weekend.

#### What should we test?

- Clickability issues round the price breakdown pie button.
- Alignment of the product description on iPhone.
- Alignment of first product and the search bar.

Full spec and testing notes: https://docs.google.com/document/d/1WZZSuMCLHiZ_UY_nLJZ8MI-HGadTQmVty7zXeS7H6VE
Design: https://app.zeplin.io/project/59532764a6c78ff3c534b372/screen/5e129ab052a96b1b1cf8916e

### CODE REVIEW

**Comment by Erioldoesdesign** [comment]
Are we missing the 'more...' link on long summaries for products? so after blueberry pie with... there should be a blue link 'more'?

Staging:
Screenshot 2020-09-20 at 23 31 10

Zeplin designs:
Screenshot 2020-09-20 at 23 31 04

File: app/assets/javascripts/templates/price_breakdown.html.haml
Comment (luisramos0) [inline app/assets/javascripts/templates/price_breakdown.html.haml:3]: is there a way we can run rubocop for haml?
I notice here that we dont have the normal spaces inside {} but we have on the line above after tt_animation

**Review by luisramos0 — APPROVED** [review]
:+1:

**Comment by mkllnk** [comment]
@Erioldoesdesign 
> Are we missing the 'more...' link on long summaries for products?

Yes, we decided to skip that part because it would add technical complexity. The ellipsis is very easy with CSS while the "more" button needs a Javascript solution. With that solution, we can then also extend the description to two lines. It's probably something we should move to another issue.

File: app/assets/javascripts/templates/price_breakdown.html.haml
Comment (mkllnk) [reply app/assets/javascripts/templates/price_breakdown.html.haml:3]: Not that I know of.

The spaces above are AngularJS syntax while the rest here is HAML syntax. I prefer no spaces within the brackets as in the [HAML reference](https://haml.info/docs/yardoc/file.REFERENCE.html) and in Ruby code.

Angular is different though. Looking at https://angularjs.org/ I see no spaces for single variable names but spaces for expressions with filters.

Comment (luisramos0) [reply app/assets/javascripts/templates/price_breakdown.html.haml:3]: the ruby style is with spaces, right?
https://github.com/rubocop-hq/ruby-style-guide#spaces-braces

Comment (mkllnk) [reply app/assets/javascripts/templates/price_breakdown.html.haml:3]: Only for blocks. Hashes can be written in both ways according to that style guide. But rubocop seems to default to spaces here. I didn't know that and always preferred no spaces. The main thing is consistency, so ... okay, I should start using spaces in hash literals and HAML...

Comment (luisramos0) [reply app/assets/javascripts/templates/price_breakdown.html.haml:3]: yeah, I thought rubocop was demanding spaces everywhere.

**Review by Matt-Yorkley — APPROVED** [review]

**Comment by daniellemoorhead** [comment]
Hey @mkllnk can this be closed and we leave #6080 as the only card to be tracked? If yes then go ahead and close this one :)

**Comment by mkllnk** [comment]
Yes, since I got the second review now, I can merge it. :+1:

**Comment by sigmundpetersen** [comment]
I guess we can remove the staging label from this one now?

**Pull request merged**
